docs/FAQ.md

```diff
@@ -49,24 +49,21 @@ You can use the `item.nbt` property. It is also recommended to use the `prismari
 **Example:**
 ```js
 function getLore(item) {
-  let message = ''
+  let message = ""
```

Comment:
Standardjs use single quotes https://standardjs.com/rules.html

Reply:
fixed it